index.html

```diff
@@ -9,37 +9,41 @@
 </head>
 <body>
     <div class="container">
-
         <div class="todo">
-            <header>
+            <header class="todo__header">
                 <h1>Todos</h1>
             </header>
 
-            <input type="text" name="new-todo" id="new-todo" placeholder="What needs to be done?">
-            <label class="small">
-                    <input type="checkbox" name="all-complete-todo" id="all-complete-todo">
-                    Mark all as complete
-            </label>
+            <div class="todo__content">
+                <input type="text" name="todo__new-todo-field" class="todo__new-todo-field" placeholder="What needs to be done?">
+
+                <div class="todo__complete-all-field checkbox-field checkbox-field--small">
```

Comment:
These 3 css classes can probably just be merged. Just go with the `todo__complete-all-field`